### PR TITLE
docs(relayer): add docs for adding relayer

### LINKED ISSUE #93: More docs on configuring the offchain worker
## Issue summary

- add more context to the README

## Other information and links

- https://github.com/ChainSafe/filecoindot#3-set-the-the-offchain-workers-account-as-relayer

### PR DESCRIPTION
**Pull request by clearloop**
## Changes

- [x] adding relayer with UI
- [x] adding relayer with polkadot.js

## Tests

```

```

## Issues

- Closes #93

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,7 +89,32 @@ the filecoindot offchain worker has been set up!
 
 #### 3. set the the offchain worker's account as relayer
 
-_TODO_
+In the provided [substrate-node-example](./substrate-node-example/README.md), we configured the [ManagerOrigin](https://github.com/ChainSafe/filecoindot/blob/5da77ea5ad329ce1e9009f5afd4f54dabbdaa6ee/substrate-node-example/runtime/src/lib.rs#L278) of `filecoindot` with `frame_system::EnsureRoot<AccountId>`, so here we need to `add_relayer` with sudo access.
```
Comment (mattsse) [inline README.md:92]: the hardcoded link will be broken eventually, I think just using `type ManagerOrigin` should be fine, without the link

File: README.md [hunk 2]
```diff
@@ -89,7 +89,32 @@ the filecoindot offchain worker has been set up!
...
+
+For setting relayer in `polkadot.js.org/apps`, we need to click `Developer -> Sudo -> filecoindot -> add_relayer` and choose an account to be the new relayer.
+
+Or with `@polkadot/api`, you can use the code below:
+
+```typescript
+import { ApiPromise, WsProvider } from "@polkadot/api";
+import { rpc, types } from "@filecoindot/types";
```
Comment (mattsse) [inline README.md:100]: this is great,
did we publish the types to npm yet? If not please open an issue to track that.

File: README.md [hunk 3]
```diff
@@ -89,7 +89,32 @@ the filecoindot offchain worker has been set up!
 
 #### 3. set the the offchain worker's account as relayer
 
-_TODO_
+In the provided [substrate-node-example](./substrate-node-example/README.md), we configured the [ManagerOrigin](https://github.com/ChainSafe/filecoindot/blob/5da77ea5ad329ce1e9009f5afd4f54dabbdaa6ee/substrate-node-example/runtime/src/lib.rs#L278) of `filecoindot` with `frame_system::EnsureRoot<AccountId>`, so here we need to `add_relayer` with sudo access.
+
+For setting relayer in `polkadot.js.org/apps`, we need to click `Developer -> Sudo -> filecoindot -> add_relayer` and choose an account to be the new relayer.
+
+Or with `@polkadot/api`, you can use the code below:
```
Comment (mattsse) [inline README.md:96]: Can we link to the polkadot-js docs, the website https://polkadot.js.org/docs/

**Review by mattsse — CHANGES_REQUESTED** [review]
great! some nits

File: README.md (hunk 2 above)
Comment (clearloop) [reply README.md:100]: not yet

**Review by mattsse — APPROVED** [review]

**Pull request merged**